### ISSUE TITLE
feature request: Custom header day name color, bigger/custom sized event circles

### ISSUE THREAD

**Issue by tryadelion** [SundeepK/CompactCalendarView#79]
may it be possible to customize the header day name colors to a different color than the text one?
also, could we be able to set bigger/custom sized event indicators?

**Comment by SundeepK** [comment]
Hi, for bigger/custom sized event indicators. Do you mean the indicators below the date or would you like a big indicator behind the current date?

**Comment by tryadelion** [comment]
below the date,  currently i'm developing an app for tablet and they look very small, around 1/5th of the textSize (which is 18sp).

**Comment by SundeepK** [comment]
Interesting, I havn't any testing with tablets, only phones. I believe the indicators should be scaled to the screen desnity though. Can you show me a screen shot if possible.

**Comment by tryadelion** [comment]
i will show it later, now i don't have the tablet with me. what about the hustom header (day names) thing? would be really cool to be able to do so.

**Comment by SundeepK** [comment]
Yep, that one is straight forward todo so I don't really have any questions for that :).

**Comment by tryadelion** [comment]
http://imgur.com/aWvGBo2

**Comment by SundeepK** [comment]
Ok thanks, will try with an emulator ans see what I can find. I don't own an android tablet and this library is based on google calendars by just eye balling things, would be nice to see what that does/looks like.
